code/modules/mob/update_icons.dm

```diff
@@ -4,6 +4,9 @@
 /mob/proc/regenerate_icons()		//TODO: phase this out completely if possible
 	return
 
+/mob/proc/update_hands()
+	return
+
```

Comment:
Are we really requed that proc?
It seems like part of code what direct access `l_hand`/`r_hand` vars.

Reply:
it's a simple wrapper for a few common parts, to save on copypasting 2-4 lines everywhere

Reply:
But why can we need call `update both hands`?
Equip/uneqip handle slot icon updates, `Item.update_wear_icon()` update current slot if icon was changed.

Reply:
You really can't see a use for updating all the icons related to someone's hands??

What if you're not sure what was changed? It's an obviously useful subset of features which is more efficient than calling regenerate icons

it sounds like your idea is better in this case, but i still believe the proc will be useful